themes/default/style.min.css

```diff
@@ -58,7 +58,7 @@ img.show-sidebar{background:url(images/bg_sprite_2.png) no-repeat 0 -22px}
 #subnavmenu a.hierarchic{padding-left:13px;background:url(images/bg_sprite_1.png) no-repeat 0 -948px}
 #subnavmenu a.fold-postings{padding-left:13px;background:url(images/bg_sprite_1.png) no-repeat 0 -998px}
 a.rss{background:url(images/bg_sprite_1.png) no-repeat 3px -1048px}
-p.right a.rss,div.small a.rss{padding-left:15px}
+p.right a.rss{padding-left:15px}
```

Comment:
What do you think about moving/adding this part to the existing rule `#footermenu li:first-child a`?

Reply:
Nice idea. When I write about a better maintainability I could be consequent and could unify these identical one-rule-rulesets because the rules are (currently) indentical. But on the other hand *IMHO* the RSS-links in `#footermenu` and the RSS-links below an entry or a thread have nothing to do with each other (HTML-structure wise) so it would be desirable to keep them separated. Additionally I would not expect to find a rule for an element of the main section in the ruleset for a link list in the page footer.

I do not feel confident with this idea. 🤔

Reply:
Okay, It was just a suggestion. ;-)

I already added your changes to my forum and it seems to work fine! Good job